pkg/publisher/publisher.go

```diff
@@ -155,7 +157,7 @@ func (p *cloudWatchPublisher) pushLocal() {
 
 func (p *cloudWatchPublisher) push(metricData []*cloudwatch.MetricDatum) {
 	if len(metricData) == 0 {
-		glog.Warning("Missing data for publishing CloudWatch metrics")
+		log.Warn("Missing data for publishing CloudWatch metrics")
```

Comment:
Is this an actionable warning? Would an admin take some action if they saw this? If not, recommend just removing it.